spec/factories/article.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :article, aliases: [:article2] do
```

Comment:
Minor question - is there a reason for the alias here?

Reply:
Defining aliases on existing factories make them easier to re-use. But in this case, I don't think it's necessary, I will just take it out.